ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -726,3 +726,7 @@ def vdot(a, b):
 
 abs = absolute
 true_divide = divide
+
+@to_ivy_arrays_and_back
+def mean(x1, /):
```

Comment:
This frontend function should have the same calling signature as the original function documented [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.mean.html)